src/pages/auth/reset-password/[token].page.tsx

```diff
@@ -44,10 +43,12 @@ const ResetPasswordPage = () => {
       if (!router.query.token) {
         router.push("/");
       } else {
-        await requestResetPassword({
+        NewResetPassword({
```

Comment:
The generated API methods in v3 are not meant to be used directly in components. Please read up on the connection system and see its uses in this codebase: https://gfw.atlassian.net/wiki/spaces/TerraMatch/pages/1423147024/Connections